tools/tracer/build.sh

```diff
@@ -0,0 +1,41 @@
+rm -rf __cmake
```

Comment:
remove this file completely. tracer should be integrated into mediasdk build system

Reply:
done